controllers/drcluster_controller.go

```diff
@@ -277,6 +272,44 @@ func filterDRClusterMCV(mcv *viewv1beta1.ManagedClusterView) []ctrl.Request {
 	}
 }
 
+func filterDRClusterSecret(reader client.Reader, secret *corev1.Secret) []ctrl.Request {
+	log := ctrl.Log.WithName("filterDRClusterSecret").WithName("Secret")
```

Comment:
I guess you used the same style as other loggers which is fine. But we probably need to find a better name for this logger. For example there is no reason to repeat "secret" twice.

We can improve this later, unifying all the filter logs to use the same style.